Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc

```diff
@@ -198,8 +202,10 @@ void DDHGCalEEAlgo::constructLayers(const DDLogicalPart& module,
 		     DDSplit(materials_[ii]).second);
       DDMaterial matter(matName);
       DDLogicalPart glog;
-      if (layerSense_[ly] == 0) {
+      if (layerSense_[ly] < 1) {
 	double alpha = CLHEP::pi/sectors_;
+	int    nsec  = (layerSense_[ly] == 0 || absorbMode_ == 0) ? 2 : 2;
```

Comment:
Is there an error here? `nsec` can be set to 2, or it can be set to 2?

Reply:
We have to take care of this in a future version. nsec will be a variable between 2 and 3. That part of the code is not yet ready and will be provided in a future version.